lib/content-services/src/lib/content-node-selector/content-node-selector.component.html

```diff
@@ -1,77 +1,82 @@
-<h1 mat-dialog-title data-automation-id="content-node-selector-title">{{title}}</h1>
-<mat-tab-group class="adf-content-node-selector-dialog-content"
-               mat-align-tabs="start"
-               (selectedIndexChange)="onTabSelectionChange($event)"
-               [class.adf-content-node-selector-headless-tabs]="!canPerformLocalUpload()">
-    <mat-tab label="{{ 'NODE_SELECTOR.REPOSITORY' | translate }}">
-            <adf-content-node-selector-panel
```

Comment:
What's happening here? it doesn't seem a style refactoring

Reply:
Yes, it's refactor of angular materials component usage. In my opinion current usage was not correct. It created a mat-tab-group, which in some cases showed only one tab and was hiding all the styling from mat-tab components. In other case it had 2 tabs and displayed everything normally. It requrired a lot of css manipulation. I think it's easier and more clear to separate those 2 situations: if required show everything in mat-tab component, else show it normally.

Reply:
I understand but as this is already a huge PR wouldn't it be possible to target this work on a separated PR?